### ISSUE TITLE
deposit: data only saved after preview

### ISSUE THREAD

**Issue by ppanero** [inveniosoftware/invenio-app-rdm#873]
Changes to the `access` works only after the preview. Changes to the settings can be submitted but are lost. There is no error message. Also, the link for data sharing requires the colleague to have an account for the Invenio instance (for me -  Sara -, that is unexpected).

**Comment by ppanero** [comment]
the reason is the fact that when going to preview it is saved, otherwise it is not. i.e. the publish button should not be active. Closing in favour of existing #794 (added to the board).
